src/ui/Windows/MainEdit.cpp

```diff
@@ -2065,6 +2065,21 @@ void DboxMain::OnGotoBaseEntry()
     CItemData *pci = getSelectedItem();
     ASSERT(pci != NULL);
 
+    if (m_bFilterActive) {
```

Comment:
As the comments state, if a filter is active so that the base entry is not in the current Tree/List view, then even though we may have removed menu the option to go to the base entry, if the user has defined an accelerator key to do this, MFC will still try to do it. This defensive code prevents failures do to the entry not being in the displays.